includes/payment-methods/class-upe-payment-method.php

```diff
@@ -476,4 +492,22 @@ private function get_currency( $order_id = null ) {
 		}
 		return get_woocommerce_currency();
 	}
+
+	/**
+	 * Check if manual renewals are accepted in WooCommerce Subscriptions settings.
+	 *
+	 * @return bool True if manual renewals are accepted in settings.
+	 */
+	private function are_manual_renewals_accepted() {
+		// Check if WooCommerce Subscriptions is active.
+		if ( ! class_exists( 'WC_Subscriptions' ) ) {
+			return false;
+		}
+
+		// Check the 'accept_manual_renewals' setting.
+		// This is the WC Subscriptions setting that controls whether manual renewals are allowed.
+		$accept_manual_renewals = get_option( 'woocommerce_subscriptions_accept_manual_renewals', 'no' );
+
+		return 'yes' === $accept_manual_renewals;
```

Comment:
There is a function for this:

```suggestion
		return function_exists( 'wcs_is_manual_renewal_enabled' ) && wcs_is_manual_renewal_enabled();
```

I think it can be used straight within `is_enabled_at_checkout()` to simplify things.